docs/Release.md

```diff
@@ -106,16 +106,18 @@ When testing the pipeline, please make sure to comment out the PMC cli commands
 
 Prerequisites:
 - Docker
-
-1. Run `generate-alpine-packaging-file.ps1` script on host computer to create `APKBUILD` file for the release. (This script can run on any Linux distro, and this script will create a docker alpine container to calculate hash keys in APKBUILD file)
-1. If you don't have account for [AlpineLinux GitLab](https://gitlab.alpinelinux.org). Create an account and [configure your SSH](https://docs.gitlab.com/ee/user/ssh.html).
-1. If you didn't fork `aports` repository yet, Fork `https://gitlab.alpinelinux.org/alpine/aports`.
-1. Clone `https://gitlab.alpinelinux.org/<your_username>/aports` repository.
-1. Navigate to `aports/community/libmsquic` folder.
-1. Replace the `APKBUILD` file with newly created `APKBUILD` file.
-1. Create a commit using `community/libmsquic: upgrade to <version_number>` (version_number e.g. 2.5.0 or 2.4.4).
-1. Create a merge request using `community/libmsquic: upgrade to <version_number>` (version_number e.g. 2.5.0 or 2.4.4).
-1. Owners of the `aports` repository will respond to the PR or merge it in couple of days/hours.
+- Powershell
+
+1. Checkout to release branch. (e.g. `git checkout v2.4.7`)
```

Comment:
This isn't a _branch_. Do you want people to checkout the branch or the tag?

Reply:
Ah, to tag, good catch thanks!